docs/DEVELOPER_GUIDE.md

```diff
@@ -193,12 +186,35 @@ this:
     └── kubectl-foo-windows.exe
 ```
 
-The resulting installation directory would up just with:
-
-```text
-.
-└── krew-foo-windows.exe
-```
+- To copy all files in the `bin/` directory of the extracted archive to
+  the root of your plugin, use the default:
+
+  ```yaml
+      files: []  # same as [{from: "*", to: "."}]
+  ```
+  
+  The resulting installation directory would eventually contain:
```

Comment:
This would copy out binaries for both platforms to the installation directory on user’s machine, despite only one of them will be used.